### DISCUSSION TITLE
Drop Down or Autocomplete for Silences Labels/Values

### DISCUSSION THREAD

**Discussion by grimz-ly** [grafana/grafana#42740] [Alerting]
If this can be done. Similar to metrics, labels, and values being autocompleted in a Prometheus query. It would be nice to have a drop down of defined labels/values or autocomplete available to choose easy when creating a silence manually in unified alerting.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
